cwf/gateway/integ_tests/assertions.go

```diff
@@ -107,6 +107,7 @@ func (tr *TestRunner) DisconnectAndAssertSuccess(imsi string) {
 func (tr *TestRunner) AssertPolicyEnforcementRecordIsNil(imsi string) {
 	recordsBySubID, err := tr.GetPolicyUsage()
 	assert.NoError(tr.t, err)
+	delete(recordsBySubID[prependIMSIPrefix(imsi)], "default_drop_flow")
```

Comment:
We should probably add a test but I guess we should do the sessiond part first, so I'll add that later